src/android/android-wizard.ts

```diff
@@ -70,8 +70,16 @@ async function runAndroidWizardWithTelemetry(
     gradle.selectAppFile(buildGradleFiles),
   );
 
-  const { selectedProject, selfHosted, sentryUrl, authToken } =
-    await getOrAskForProjectData(options, 'android');
+  const projectData = await getOrAskForProjectData(options, 'android');
+
+  if (projectData.spotlight) {
+    clack.log.warn('Spotlight mode is not yet supported for Android.');
+    clack.log.info('Spotlight is currently only available for Next.js.');
+    await abort('Exiting wizard', 0);
+    return;
+  }
```

Comment:
for all wizard not supporting spotlight yet, we return some logs and exit the wizard in case someone runs the wizard with `--spotlight`.

wdyt @Lms24 ? we're going to add support for the other ones, but until then. 

This is necessary as we now do the union type based on spotlight = true/false as you suggested